lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -86,6 +89,27 @@ def replacement_text(range)
           end
         end
 
+        def text(node)
+          if node.dstr_type?
```

Comment:
generally, if you move the `join` to the result of `flatten_node`, so it always returns a string, you can remove this if/else part

Reply:
Great tip!
It's so much better with one method less 👍